test/libyul/yulOptimizerTests/reasoningBasedSimplifier/addmod.yul

```diff
@@ -9,6 +9,8 @@
 
     // addmod is equal to mod of sum for small numbers
     if and(and(lt(x, 1000), lt(y, 1000)), lt(z, 1000)) {
+        // z3 <4.8.10 was able to infer that the
```

Comment:
Which was the only reason for this test file... ;)

Reply:
Yea.. well, we can keep it for reference